frontend/packages/console-shared/src/hooks/__tests__/usePinnedResources.spec.ts

```diff
@@ -52,6 +60,7 @@ describe('usePinnedResources', () => {
   });
 
   it('returns no default pins if there are no other pins defined and no extension could be found', async () => {
+    window.SERVER_FLAGS.perspectives = '[{ "id" : "dev", "visibility": {"state" : "Enabled" }}]';
```

Comment:
Here as well. If you want you can duplicate this test to check what happen to pinned resources when its not configured, Enabled, Disabled. But for this first test cases I would keep "no customization"

```suggestion
    window.SERVER_FLAGS.perspectives = '';
```